sdk/sdk-api/src/main/java/mil/emp3/api/utils/kml/ZipUtils.java

```diff
@@ -0,0 +1,87 @@
+package mil.emp3.api.utils.kml;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+public class ZipUtils {
+
+    private List <String> fileList;
+    private static String OUTPUT_ZIP_FILE = "Folder.zip";
+    private static String SOURCE_FOLDER = "D:\\Reports"; // SourceFolder path
+
+    public ZipUtils() {
+        fileList = new ArrayList < String > ();
+    }
+
+    public static void callZip(String sourceFolder, String outputFolder) {
+        OUTPUT_ZIP_FILE = outputFolder;
+        SOURCE_FOLDER = sourceFolder;
+        ZipUtils appZip = new ZipUtils();
+        appZip.generateFileList(new File(SOURCE_FOLDER));
+        appZip.zipIt(OUTPUT_ZIP_FILE);
+    }
+
+    public void zipIt(String zipFile) {
+        byte[] buffer = new byte[1024];
+        String source = new File(SOURCE_FOLDER).getName();
+        FileOutputStream fos = null;
+        ZipOutputStream zos = null;
+        try {
+            fos = new FileOutputStream(zipFile);
+            zos = new ZipOutputStream(fos);
+
+            System.out.println("Output to Zip : " + zipFile);
+            FileInputStream in = null;
+
+            for (String file: this.fileList) {
+                System.out.println("File Added : " + file);
```

Comment:
same here, remove print statements

Reply:
If you want to log the file names, use the Log.d or Log.i methods.